### PR TITLE
Add mobile ticks

### PR DESCRIPTION
**Pull request by lubej**

### CODE REVIEW

**Comment by lubej** [comment]
Task:
https://app.clickup.com/t/861mx1fuv

File: src/index.tsx [hunk 1]
```diff
@@ -10,6 +10,9 @@ import './styles/index.css'
 import './locales/i18n'
 import Axios from 'axios'
 
+// ES2020 Intl.NumberFormatOptions Polyfill
+import '@formatjs/intl-numberformat/polyfill'
```
Comment (buberdds) [inline src/index.tsx:14]: why we need polyfill ?

File: src/app/components/charts/LineChart.tsx [hunk 2]
```diff
@@ -57,6 +60,9 @@ const LineChartCmp = <T extends object>({
         tick={withLabels ? { fill: COLORS.brandDark, strokeWidth: 0 } : false}
         type="number"
         tickMargin={tickMargin}
+        tickFormatter={tick => {
+          return tickMark ? intlNumberFormat(tick) : tick
```
Comment (buberdds) [inline src/app/components/charts/LineChart.tsx:64]: can't we handle this via i18next?

Comment (csillag) [reply src/app/components/charts/LineChart.tsx:64]: I think we can handle this with bignumber. For now, let's just  use `number.toLocaleString()`, and add a TODO, and I will pulish it in #333.

Comment (buberdds) [reply src/app/components/charts/LineChart.tsx:64]:
```
          tickFormatter={tick =>
            t('foo', {
              value: tick,
              formatParams: {
                value: {
                  notation: 'compact',
                } satisfies Intl.NumberFormatOptions,
              },
            })
          }
          ```
          in lang file `"foo": "{{value, number}}",`

Comment (lubej) [reply src/app/components/charts/LineChart.tsx]: Right, thanks.  TypeScript was complaining for some reason, and I was thinking we need the polyfill. Removed.

File: src/index.tsx (hunk 1 above)
Comment (lubej) [reply src/index.tsx]: Removed

**Review by buberdds — APPROVED** [review]

**Pull request merged**